src/interfaces/product.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { ProductType } from '../enums/product-type.enum'
+
+export interface CreateProduct {
+  name: string
+  description: string
+  unit_price: number
```

Comment:
I've added a comment in `src/models/product.model.ts` regarding the use of `snake_case`. You can refer to that comment for more details.